index.html

```diff
@@ -3,9 +3,19 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Home</title>
+    <link rel="stylesheet" href="./styles/index.css">
+    <link rel="stylesheet" href="./styles/general.css">
+    <link rel="stylesheet" href="./styles/variables.css">
+    <link rel="stylesheet" href="./styles/Fonts/KoHo_WOFF/KoHo-Bold.woff2">
```

Comment:
Here we can link the styleheet of Koho instead. so we get access to whole "font face library" automatically

Reply:
ok